app/controllers/shopify_app/extension_verification_controller.rb

```diff
@@ -2,19 +2,14 @@
 
 module ShopifyApp
   class ExtensionVerificationController < ActionController::Base
+    include ShopifyApp::PayloadVerification
     protect_from_forgery with: :null_session
     before_action :verify_request
 
     private
```

Comment:
@adfaris Thanks for restoring here - I think we should also add the `private` declaration in `lib/shopify_app/controller_concerns/payload_verification.rb` so that `shopify_hmac` and `hmac_valid?` are private.

Reply:
@andyw8 Thanks for the feedback and those methods are private now.